aptos-move/block-executor/src/executor_utilities.rs

```diff
@@ -76,6 +79,28 @@ macro_rules! resource_writes_to_materialize {
 pub(crate) use groups_to_finalize;
 pub(crate) use resource_writes_to_materialize;
 
+// Returns the tags written to (and stored) by the previous incarnation, if the KeyKind is a Group,
+// otherwise None. If the kind does not match, PanicError is returned.
```

Comment:
nit: make it documentation vi `///`? Also you can call it as `remove_from_previous_group_keys` if groups only are expected, but up to you